user-contracts-mon/internal/monitor.go

```diff
@@ -0,0 +1,146 @@
+package monitor
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/NicoNex/echotron/v3"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/deployer"
+	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/graphql"
+)
+
+// Monitor struct of parsed configration
+type Monitor struct {
+	Bot      echotron.API
+	Mnemonic string
+	BotToken string
+	Network  string
+	interval int
+}
+
+// NewMonitor creates a new monitor from parsed config/env file
+func NewMonitor(conf Config) (Monitor, error) {
+	mon := Monitor{}
+	mon.Mnemonic = conf.mnemonic
+	mon.Network = conf.network
+	mon.interval = conf.interval
+	mon.BotToken = conf.botToken
+	mon.Bot = echotron.NewAPI(mon.BotToken)
+
+	return mon, nil
```

Comment:
it can't return an error, should it return `(Monitor, error)` or just `Monitor`?